fmriprep/workflows/bold/resampling.py

```diff
@@ -113,7 +112,8 @@ def select_target(subject_id, space):
                          mem_gb=DEFAULT_MEMORY_MIN_GB)
     resampling_xfm = pe.Node(LTAConvert(in_lta='identity.nofile', out_lta=True),
                              name='resampling_xfm')
-    set_xfm_source = pe.Node(ConcatenateLTA(out_type='RAS2RAS'), name='set_xfm_source')
+    merge_xfm = pe.Node(niu.Merge(2), name="merge_xfm", run_without_submitting=True)
+    concat_xfm = pe.Node(ConcatenateXFMs(out_fmt="fs"), name="concat_xfm")
```

Comment:
I think it's left empty because the `filename =` metadata is not set. But there's no failsafe for that in nitransforms right now, so one could think of setting it and getting bitten by this.